cwms-data-api/src/test/resources/cwms/cda/data/dto/timeseriesprofile/timeseriesprofileinstance.json

```diff
@@ -1,42 +1,48 @@
 {
   "time-series-profile":
   {
-    "location-id": "DET",
+    "location-id": {
+      "office-id": "office",
+      "name": "location"
+    },
     "key-parameter": "Depth",
-    "parameter-list": [ "Temp-Water", "Depth"],
-    "description": "Demonstration Profile for Couger Lake",
-    "office-id": "SWT",
-    "ref-ts-id": "DET.Elev-Forebay.Inst.0.0.Mixed-Rev"
+    "parameter-list": [ "Temperature", "Depth"],
+    "description": "Description",
+    "reference-ts-id":{
+      "office-id": "office",
+      "name": "location.Elev.Inst.0.0.REV"
+    }
   },
-
-  "timeseries-list" :  [
+  "time-series-list" :  [
     {
-         "parameter": "Temp-Water",
+         "parameter": "Temperature",
```

Comment:
? The original was correct. Temperature is not a base parameter in CWMS.... or have I gone mad?

Reply:
switched to Temp-Water again